### ISSUE TITLE
Plex managed user sees all libraries, even ones he should not

### ISSUE THREAD

**Issue by croneter** [croneter/PlexKodiConnect#646]
First, some background:
I have 2 plex users: cyriltra (plex admin), Public User.
cyriltra : has access to all libraries
Public User : has access to 3 libraries: Movies, TV Shows, Music

With Kodi & PKC, im using Public User

The issue
After the auto upgrade of PKC from 2.5.23 to 2.6 => I realize Kodi is now displaying all libraries.
I checked the settings and the logged in user was now cyriltra, instead of Public User

1. then manually log out cyriltra (via PKC settings) and decided to log in again.
2. Here, PKC gave me the plex link code to input in plex.tv/link, which i did
3. once the account is linked, PKC asked me to select my plex user: cyriltra or public user
4. i select public user and PKC provided me another plex link code.
5. I input as well this new code in plex.tv/link
6. Once the account is linked again, PKC does not asked me to select a plex user again.
7. when I go to PKC settings, the plex user used is cyriltra and not public user
8. For the moment, I dont have any debug log, but I will try to provide you asap. Meanwhile, you can maybe try to reproduce these steps and see if it’s really a bug or not.

I wrote in this forum 16 days ago about this issue. I was away from home the entire December and until November i never encountered this issue.
early Jan, I fixed this issue by logging out and login in several time, which at some point PKC starts using Public User
I then updated PKC (beta) from 2.5.17 to 2.5.23 without any further issue. Then yesterday it automatically upgraded to 2.6 and the issue came back

**Comment by croneter** [comment]
Switching from an admin user to a Plex managed user, I even get endless library sync error messages and `Deleting` pop-ups

**Comment by Cyriltra** [comment]
Hi Croneter,

thanks for the quick fix. I gave it a try by: 
1) logging out my current plex user (cyriltra)
2) then logging in => plex.tv/link
3) selection of plex user: Public User

then I got the message "Welcome Public User" and the content is related to Public User (not cyriltra). So it seems to be ok.
However, there is 1 last issue: in PKC Settings -> Plex -> Plex user stills shows cyriltra (not Public User). Still same after reboot.

Thank you
